code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -209,8 +209,12 @@
 /datum/reagent/water/holywater/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
 	..()
 	if(ishuman(M)) // Any location
-		if(M.mind && cult.is_antagonist(M.mind) && prob(10))
-			cult.remove_antagonist(M.mind)
+		if(iscultist(M))
+			if(prob(10))
+				cult.offer_uncult(M)
+			if(prob(2))
+				var/obj/effect/spider/spiderling/S = new /obj/effect/spider/spiderling(M.loc)
```

Comment:
I wasn't suggesting that they'd use it to grief the station so much as an effective strat.

That's not the only problem though; it's going to also cause them to spawn in nullspace if they're inside non-closets....which is more or less a memory leak.